<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Distance Matrix</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Input point layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inPoint1"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Input unique ID field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inField1"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Target point layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inPoint2"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Target unique ID field</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inField2"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output matrix type</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rdoLinear">
        <property name="text">
         <string>Linear (N*k x 3) distance matrix</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rdoStandard">
        <property name="text">
         <string>Standard (N x T) distance matrix</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="rdoSummary">
        <property name="text">
         <string>Summary distance matrix (mean, std. dev., min, max)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="chkNearest">
        <property name="text">
         <string>Use only the nearest (k) target points</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spnNearest">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output distance matrix</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="outFile">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFile">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNearest</sender>
   <signal>toggled(bool)</signal>
   <receiver>spnNearest</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoStandard</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkNearest</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rdoStandard</sender>
   <signal>toggled(bool)</signal>
   <receiver>spnNearest</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
